docs/getting-started-9c-with-the-unity-editor/getting-started-9c-with-the-unity-editor.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting started Nine Chronicles with the Unity editor
+---
+
+Planetarium에서 배포하는 Nine Chronicles(이하 9C)는 유니티 엔진을 사용해서 만들었습니다. 이는 유니티 에디터로 실행할 수 있다는 말이고, 이 문서에서는 그 방법을 소개합니다.
+
+# Install Unity Engine
+
+여기서는 유니티 허브를 사용해서 유니티 엔진을 설치하는 방법을 소개합니다.
+
+## Install Unity Hub
+
+### 1. [유니티 허브 다운로드 링크][unity-hub-download-link]를 클릭합니다.
+
+- 위의 링크가 동작하지 않으면 [유니티 다운로드 페이지][unity-download-page]에서 유니티 허브를 다운로드 합니다.
+
+### 2. 다운로드한 유니티 허브 설치 파일을 실행합니다.
+
+[unity-hub-download-link]: https://public-cdn.cloud.unity3d.com/hub/prod/UnityHubSetup.dmg
+[unity-download-page]: https://unity3d.com/get-unity/download
+
+## Install Unity Engine(2021.3.5f1)
+
+### 1. [유니티 허브용 유니티 엔진(2021.3.5f1) 설치 링크][unity-install-with-hub-2021.3.5f1]를 클릭합니다.
+
+- 위 링크가 동작하지 않으면 [유니티 다운로드 아카이브 페이지][unity-download-archive-page]에서 유니티 엔진(2021.3.5f1)을 다운로드 합니다.(유니티 허브 버튼으로 설치하기를 권장)
+
+### 2. 앞서서 유니티 허브를 설치했다면, 유니티 허브가 실행되면서 2021.3.5f1 버전 유니티 엔진의 설치가 진행됩니다.
+
+[unity-install-with-hub-2021.3.5f1]: unityhub://2021.3.5f1/40eb3a945986
+[unity-download-archive-page]: https://unity3d.com/get-unity/download/archive
+
+# Clone the Nine Chronicles repository
+
+여기서는 저장소를 복제하는 방법으로 [Git CLI][git-page]를 사용하는 방법과 [Fork][fork-page]를 사용하는 방법을 다룹니다.
+
+[git-page]: https://git-scm.com/
+[fork-page]: https://git-fork.com/
+
+## Clone with Git CLI
+
+### 1. 운영체제에 맞는 Git CLI를 설치합니다.
+
+- [Windows](https://git-scm.com/download/windows)
+- [Mac](https://git-scm.com/download/mac)
+- [Linux](https://git-scm.com/download/linux)
+
+### 2. 9C를 복제합니다.
+
+```
+$ git clone https://github.com/planetarium/NineChronicles.git
+$ git submodule update --init --recursive
+$ git config core.hooksPath hooks
+```
+
+### 3. main 브랜치로 체크아웃합니다.
+
+```
+$ git checkout main
+$ git submodule update --recursive
+```
+
+## Clone with Fork
+
+### 1. 운영체제에 맞는 Fork 설치합니다.
+
+- [Windows(installer)](https://git-fork.com/update/win/ForkInstaller.exe)
+- [Mac(2.20.2)](https://cdn.fork.dev/mac/Fork-2.20.2.dmg)
+- 위 링크가 동작하지 않거나 다른 버전을 설치하고 싶다면 [Fork 페이지][fork-page]로 접속해서 다운로드 합니다.
+
+### 2. Fork를 실행해서 저장소를 Clone 합니다.
+
+- Fork 메뉴: File > Clone
+
+  <img
+    src="./images/Screen%20Shot%202022-08-30%20at%2011.48.24%20AM.png"
+    style="width:50vw" />
```

Comment:
Do you have any reason not using MD style but HTML style? I know this works on markdown but I think we can use same format to embed images.
Or please add alt text at least.

Reply:
@U-lis I wanted to resize the image. OK I'll add alt text.